src/sentry/api/endpoints/project_metrics_extraction_rules.py

```diff
@@ -60,16 +61,17 @@ def get(self, request: Request, project: Project) -> Response:
             return Response(status=404)
 
         try:
-            extraction_rules = get_metrics_extraction_rules(project)
+            configs = SpanAttributeExtractionRuleConfig.objects.filter(project=project)
+
         except Exception as e:
             return Response(status=500, data={"detail": str(e)})
 
         return self.paginate(
             request,
-            queryset=extraction_rules,
+            queryset=list(configs),
```

Comment:
Nit for future work: We should think of using different pagination in the future, but this is currently out of the scope of this PR.

How we currently doing this, we will fetch all the configs from the database, and then slice the list of the fetched configs and return only part of that list. Instead we should do the pagination during querying the database (relay on OFFSET and LIMIT from Postgres)

Reply:
Added a TODO comment for this